recipe/meta.yaml

```
@@ -1,58 +1,85 @@
 {% set name = "tobler" %}
-{% set version = "0.5.4" %}
+{% set version = "0.12.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pysal/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 2c902fe2b708b9d6e8bbf461c25b8b9eb0d70f52ea88f560ace6a1a4267a5965
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 96a05c2bd69569c5fadb940f7ad242dd8a43d12c7818c7ab97534e46670677f1
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  # Skip Python <3.10 due to libpysal <4.10 (noarch) causing import errors with scipy >=1.8.0
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
-    - python 3.9
+    - python
     - pip
     - setuptools
```

```suggestion
    - setuptools
    - setuptools_scm >=6.2
```
https://github.com/pysal/tobler/blob/v0.12.1/pyproject.toml#L2C34-L2C59